content/en/serverless/guide/azure_app_service_linux_sidecar.md

```diff
@@ -0,0 +1,170 @@
+
+#### Full example Dockerfile
+
+```dockerfile
+# Stage 1: Build the application
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build
+WORKDIR /app
+
+# Copy the project file and restore dependencies
+COPY *.csproj ./
+RUN dotnet restore
+
+# Copy the remaining source code
+COPY . .
+
+# Build the application
+RUN dotnet publish -c Release -o out
+
+# Stage 2: Create a runtime image
+FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS runtime
+WORKDIR /app
+
+# Copy the build output from stage 1
+COPY --from=build /app/out ./
+
+# Datadog specific
+RUN mkdir -p /datadog/tracer
+RUN mkdir -p /home/LogFiles/dotnet
+
+ADD https://github.com/DataDog/dd-trace-dotnet/releases/download/v2.49.0/datadog-dotnet-apm-2.49.0.tar.gz /datadog/tracer
+RUN cd /datadog/tracer && tar -zxf datadog-dotnet-apm-2.49.0.tar.gz
+
+# Set the entry point for the application
+ENTRYPOINT ["dotnet", "<your dotnet app>.dll"]
+```
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+Then, build the image and push it to your preferred container registry.
+
+### Create your Linux Web App
+
+1. In the Azure Portal, go to **App Services** and select **Create**.
+1. On the **Basics** tab, provide the required details.
+   - For **Publish**, select **Container**.
+   - For **Operating System**, select **Linux**.
+   Then, select **Next: Container >**.
+1. On the **Container** tab, provide the required details.
+   - For **Sidecar support**, select **Enabled**.
+   - For **Image Source**, select your chosen registry.
+   - Then, specify the **Registry**, **Image**, **Tag**, and **Port** for your container image.
+1. Select **Review + create**, then select **Create**.
+
+### Add Datadog environment variables
+
+In the Azure portal, select your app. In the left menu, select **Configuration** > **Application settings**. Then, add the following environment variables as application settings.
+
+
+`DD_API_KEY` 
+: Your [Datadog API key][3]. <br/>
+**Required**. Alternatively, you can source your API key (and other sensitive information) from Azure Key Vault. See [Use Key Vault references as app settings in Azure App Service][4].
+
+`DD_SITE` 
+: {{< region-param key="dd_site" code="true" >}} <br/>
+Corresponds to your [Datadog site][5]. Defaults to `datadoghq.com`.
+
+`DD_SERVICE` 
+: Supply a service name to be displayed in your Datadog [Service Catalog][6]. See [Unified Service Tagging][7]. 
+
+`DD_ENV` 
+: A name for your environment, such as `staging` or `prod`. See [Unified Service Tagging][7].
+
+`DD_SERVERLESS_LOG_PATH` 
+: `/home/Logfile/*.log` <br/>
+Corresponds to the path where you write your application logs. If you have changed this location, specify your custom location in this setting.
+
+`DD_DOTNET_TRACER_HOME` 
+: `/datadog/tracer`
+
+`DD_TRACE_LOG_DIRECTORY` 
+: `/home/Logfiles/dotnet`
+
+`CORECLR_ENABLE_PROFILING` 
+: `1`
+
+`CORECLR_PROFILER` 
+: `{846F5F1C-F9AE-4B07-969E-05C26BC060D8}`
+
+`CORECLR_PROFILER_PATH` 
+: `/datadog/tracer/Datadog.Trace.ClrProfiler.Native.so`
```

Comment:
I missed something last time. We should mark these as required since the tracer needs them to start. I don't want customers to not set them and wonder why the tracer isn't working.